infra/base-images/base-runner/coverage

```diff
@@ -97,16 +97,18 @@ function run_fuzz_target {
 
   shared_libraries=$(coverage_helper shared_libs -build-dir=$OUT -object=$target)
 
-  local summary_only_flag="-summary-only"
+  llvm-cov export -summary-only -instr-profile=$profdata_file -object=$target \
+      $shared_libraries $LLVM_COV_COMMON_ARGS > $FUZZER_STATS_DIR/$target.json
+
   if [ -n "${FULL_SUMMARY_PER_TARGET-}" ]; then
     # This is needed for dataflow strategy analysis, can be removed later. See
     # - https://github.com/google/oss-fuzz/pull/3306
     # - https://github.com/google/oss-fuzz/issues/1632
-    summary_only_flag=""
+    # Intentionally writing these to the logs dir in order to hide the dumps
+    # from the ClusterFuzz cron job.
+    llvm-cov export -instr-profile=$profdata_file -object=$target \
+      $shared_libraries $LLVM_COV_COMMON_ARGS > $LOGS_DIR/$target.json
```

Comment:
didn't want to invent a new dir for this, as it has to be separately uploaded (via a separate GCB step -- too much trouble)